### PR TITLE
Make the browserslist explicit

### PR DESCRIPTION
**Pull request by cquiroz**
Signed-off-by: Carlos Quiroz <carlos.quiroz@noirlab.edu>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,5 +72,10 @@
     "fix-light": "sed '/^[\\.\\}\\@]/d;/^[[:blank:]]*\\..*/d;/\\/\\/.*/d' common/src/main/webapp/less/variables-light.less > vars.css;trap \"rm vars.css\" EXIT;npx stylelint --fix common/src/main/webapp/less",
     "build": "node --unhandled-rejections=strict build.js",
     "serve": "vite preview"
-  }
+  },
+  "browserslist": [
```
Comment (rpiaggio) [inline package.json:76]: When is this used? I'm suspecting stylelint?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (cquiroz) [reply package.json:76]: Yes, but more important it is used to do auto prefixing, anyway I saw a warning that the list should rather be explicit

Comment (toddburnside) [reply package.json:76]: I had to do a search for what this was about. Came up with https://stackoverflow.com/questions/55510405/what-is-the-significance-of-browserslist-in-package-json-created-by-create-react

**Pull request merged**